src/codegen.cpp

```diff
@@ -4146,6 +4146,12 @@ static LLVMValueRef ir_render_frame_address(CodeGen *g, IrExecutable *executable
     return LLVMBuildCall(g->builder, get_frame_address_fn_val(g), &zero, 1, "");
 }
 
+static LLVMValueRef ir_render_handle(CodeGen *g, IrExecutable *executable,
```

Comment:
@andrewrk I am pretty sure that the `coro_handle` is stored in `IrExecutable`, but I am wondering how to expose it as an `LLVMValueRef` ? It seems to have no `llvm_value`.

Reply:
Solved it by `LLVMBuildBitCast`ing `executable->fn_entry->ir_executable.coro_handle->other`

Reply:
Hmm. That might be ok. A more straightforward solution would probably be to use `@llvm.coro.frame`: http://llvm.org/docs/Coroutines.html#llvm-coro-frame-intrinsic

Reply:
To be clear, I think you can leave this as is, but if it causes problems, we'll know to try this other approach.

Reply:
Cool, I replaced it with the `llvm.coro.frame` which I think will give better separation between IR and codegen. I did look through the docs several times, and I am not sure why I didn't pickup on it. Thanks!